RetakesAllocatorCore/Config/Configs.cs

```diff
@@ -21,6 +21,7 @@ public static class Configs
         WriteIndented = true,
         AllowTrailingCommas = true,
         ReadCommentHandling = JsonCommentHandling.Skip,
+        DefaultIgnoreCondition = JsonIgnoreCondition.Never
```

Comment:
This is already the default value per https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions.defaultignorecondition?view=net-7.0 so I don't think its changing anything